nodes/KlickTipp/actions/field/index.ts

```diff
@@ -1,8 +1,9 @@
 import type { INodeProperties } from 'n8n-workflow';
 
 import * as index from './index.operation';
+import * as get from './get.operaton';
```

Comment:
issue: when correcting the file name change here, too

Reply:
Fixed